packages/cli/src/ui/hooks/atCommandProcessor.ts

```
@@ -494,8 +500,13 @@ export async function handleAtCommand({
       >,
       userMessageTimestamp,
     );
-    return { processedQuery: null, shouldProceed: false };
+    const firstError = resourceReadDisplays.find(
```

any reason why only first error is returned and discarding all the other errors?